Library/Formula/hostdb.rb

```diff
@@ -1,14 +1,23 @@
-require 'formula'
-
 class Hostdb < Formula
   desc "Generate DNS zones and DHCP configuration from hostlist.txt"
-  homepage 'http://code.google.com/p/hostdb/'
-  url 'https://hostdb.googlecode.com/files/hostdb-1.004.tgz'
-  sha1 '65ec59c2c88b763813fa611d8fd28a45cd9d5278'
+  homepage "https://code.google.com/p/hostdb/"
+  url "https://hostdb.googlecode.com/files/hostdb-1.004.tgz"
+  sha256 "beea7cfcdc384eb40d0bc8b3ad2eb094ee81ca75e8eef7c07ea4a47e9f0da350"
+
+  def examples_path
+    share + "examples" + name
+  end
```

Comment:
You may change your mind again after seeing what had to be done to keep it passing `brew audit --strict`. Just let me know. :smile_cat: